### PR TITLE
KRNL-271 update TranslationService

### PR DESCRIPTION
**Pull request by Dosexe**
+ TranslationRetrieveOptions for Retrieve call
* IsFuzzy -> IsUnverified
+ SegmentNumber to Translation
* FilterLangID string -> int64

### CODE REVIEW

File: svc_translation.go [hunk 1]
```diff
@@ -30,11 +32,12 @@ type Translation struct {
 	ModifiedAtTs    int64  `json:"modified_at_timestamp"`
 	ModifiedBy      int64  `json:"modified_by"`
 	ModifiedByEmail string `json:"modified_by_email"`
-	IsFuzzy         bool   `json:"is_fuzzy"`
```
Comment (yarlson) [inline svc_translation.go:33]: Breaking changes

File: svc_translation.go [hunk 2]
```diff
@@ -111,7 +114,7 @@ func (t *NewTranslation) UnmarshalJSON(data []byte) error {
 
 type UpdateTranslation struct {
 	Translation                string   `json:"translation"`
-	IsFuzzy                    *bool    `json:"is_fuzzy,omitempty"`
```
Comment (yarlson) [inline svc_translation.go:114]: Breaking changes

File: svc_translation.go [hunk 3]
```diff
@@ -222,10 +225,12 @@ type TranslationListOptions struct {
 	// Possible values are 1 and 0.
 	DisableReferences uint8 `url:"disable_references,omitempty"`
 
-	FilterLangID     string `url:"filter_lang_id,omitempty"`
-	FilterIsReviewed uint8  `url:"filter_is_reviewed,omitempty"`
-	FilterFuzzy      uint8  `url:"filter_fuzzy,omitempty"`
-	FilterQAIssues   string `url:"filter_qa_issues,omitempty"`
+	FilterLangID       int64  `url:"filter_lang_id,omitempty"`
```
Comment (yarlson) [inline svc_translation.go:228]: Breaking changes

File: svc_translation.go (hunk 1 above)
Comment (Dosexe) [reply svc_translation.go:33]: Yes, because the current version is invalid and won't work with Lokalise Api

File: svc_translation.go (hunk 2 above)
Comment (Dosexe) [reply svc_translation.go:114]: Yes, because the current version is invalid and won't work with Lokalise Api

File: svc_translation.go (hunk 3 above)
Comment (Dosexe) [reply svc_translation.go:228]: Yes, because the current version is invalid and won't work with Lokalise Api

**Pull request merged**